keymanapp/keyman
# chore(common): Check in crowdin strings for Hausa

Checks in crowdin strings for Hausa.

## User Testing
For all the platforms, load the PR build, set the UI to "Hausa", and verify the UI is in Hausa

* **TEST_ANDROID** - Tests Hausa UI for Android
1. Load the PR build in an Android 5.0 emulator (SDK 21)
2. In the Keyman app, go to Settings --> Display Language --> Hausa
3. Verify the UI is in Hausa

* **TEST_IOS** - Tests Hausa UI for iOS
This won't be available to test until @sgschantz runs the xcode project to add Hausa (language code `ha`)

* **TEST_WINDOWS** - Tests Hausa UI for Windows

* **TEST_LINUX** - Tests Hausa UI for Linux
